bin/lang/german.json

```
@@ -0,0 +1,105 @@
+"IDC_CHECK_OUTPUT_NO_OVERWRITE":"Ausgabedateien nicht überschreiben",
+"MessageUseProcessorMode":"Genutzter Prozessor: %s",
+"MessageProcessTime":"Umwandlungszeit: %02d:%02d:%02d.%03d",
+"MessageInitTime":"Initialisierungszeit: %02d:%02d:%02d.%03d",
+"MessagecuDNNCheckTime":"cuDNN-überprüfungszeit: %02d:%02d:%02d.%03d",
+"MessageTitleError":"Fehler",
+"MessageScaleRateCheckError":"Vergrößerungsfaktor muss größer als 0.0 und positiv sein.",
+"MessageScaleWidthCheckError":"Größe muss eine Ganzzahl und größer als 0 sein.",
+"MessageCropSizeCheckError":"Teilungsgröße muss größer als 0 sein.",
+"MessageInputPathCheckError":"Bitte gebe einen Eingabepfad an.",
+"MessageOutputPathCheckError":"Bitte gebe einen Ausgabepfad an.",
+"MessageOutputExtCheckError":"Bitte eine Dateiendung für die Ausgabe wählen.",
+"MessageInputCheckError":"Eingabedatei oder Ordner existiert nicht.",
+"MessageOutputExtentionCheckError":"Bitte die Dateiendung der Ausgabe wählen.",
+"MessageOutputQualityCheckError":"Ausgabequalität ist außerhalb des gültigen Bereichs",
+"MessageOutputDepthCheckError":"Bittiefe der Ausgabe ist inkorrekt.",
+"MessageCudaNotFindError":"Eingabe kann nicht mittels GPU umgewandelt werden.\r\nDer CUDA Treiber ist möglicherweise nicht installiert. \r\nInstalliere bitte den CUDA Treiber.",
+"MessageCudaOldVersionError":"Eingabe kann nicht mittels GPU umgewandelt werden.\r\nDie CUDA Treiberversion scheint veraltet.\r\nBitte aktualisiere den CUDA Treiber.",
+"MessageTransSuccess":"Erfolgreich umgewandelt",
+"MessageErrorHappen":"Ein Fehler ist aufgetreten",
+"MessageCreateOutDirError":"Ausgabeordner\r\n[%s]\r\nkonnte nicht erstellt werden",
+"MessageCancelError":"Umwandlung wurde abgebrochen",
+"MessageInvalidParameterError":"Parameter ist ungültig",
+"MessageFailedOpenModelFileError":"Modeldatei konnte nicht geöffnet werden",
+"MessageFailedParseModelFileError":"Modeldatei konnte nicht verarbeitet werden",
+"MessageFailedConstructModelError":"Model konnte nicht konstruiert werden",
+"MessageFailedOpenInputFileError":"Die Eingabedatei %s konnte nicht geöffnet werden",
+"MessageFailedOpenOutputFileError":"In die Ausgabedatei %s konnte nicht geschrieben werden",
+"MessageFailedProcessCaffeError":"Die Interpolation konnte nicht durchgeführt werden",
+"MessageTitleResult":"Ergebnis",
+"MessagecuDNNOK":"cuDNN kann genutzt werden",
+"MessagecuDNNNotFindError":"cuDNN kann nicht genutzt werden\r\n[%s] konnte nicht gefunden werden",
+"MessagecuDNNOldVersionError":"cuDNN kann nicht genutzt werden\r\n[%s] ist eine veraltete Version. Bitte v2 benutzen",
+"MessagecuDNNCannotCreateError":"cuDNN kann nicht genutzt werden\r\ncKonnte cuDNN nicht initialisieren",
+"MessagecuDNNDefautlError":"cuDNN kann nicht genutzt werden",
+"MessageExtStr":"Eingabedateiendungen(%s)",
+"MessageAllFileFolder":"Alle Dateien und Ordner(*.*)",
+"MessageTitleInputDialog":"Bitte wähle eine Datei oder einen Ordner",
+"MessageTimeLeftUnknown":"Übrige Zeit: Unbekannt",
```

Should be better "Restzeit" or "Restdauer".
```suggestion
"MessageTimeLeftUnknown":"Restzeit: Unbekannt",
```